lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -26,6 +25,16 @@ def initialize(payload)
       @payload = payload
     end
 
+    def description
+      if format == "policy"
```

Comment:
NOTE FOR THIS PR, but if we have time at the end of the quarter, it would be nice to try and remove the description where it is just a duplicate of content in the `indexable_content` field and see if it effect the score in any way.

That would enable us to remove this mess, plus some indexable content hacks.